### PR TITLE
Fixed two @everyone injections, a homegrown clean_content and, worst of all, a grammar error 0.o

### PR DESCRIPTION
**Pull request by codingJWilliams**
Yes, I did go through every command's source

### CODE REVIEW

File: cogs/meta.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def __init__(self, bot):
                              or self.bot.get_command(command)
 
                     if entity is None:
-                        clean = command.replace('@', '@ ')  # non breaking space
+                        clean = commands.clean_content().convert(command)  # non breaking space
```
Comment (perryprog) [inline cogs/meta.py]: Remove comment here

File: cogs/other.py [hunk 2]
```diff
@@ -483,7 +483,7 @@ def __init__(self, bot):
                             )
                             return await ctx.send(
                                 '\n'.join(
-                                    f'**{r[0].display_name}:** {r[1]}'
```
Comment (perryprog) [inline cogs/other.py]: Line length

File: cogs/tracking.py [hunk 3]
```diff
@@ -44,7 +44,7 @@ def __init__(self, bot):
         await self.send_guild(discord.Embed(color=discord.Color.red()), guild)
 
     async def on_command_error(self, ctx, error):
-        # must be tulpa
```
Comment (perryprog) [inline cogs/tracking.py]: Change to might be tulple

**Comment by perryprog** [comment]
Denying for nsfw language in commit messages, please recreate.

**Comment by codingJWilliams** [comment]
wow

**Pull request closed without merging**